Week-3/Exercise-3.3.js

```diff
@@ -0,0 +1,30 @@
+function createIncrement() {
+    
+    let count = 0;
+    
+    function increment() {
+        count++;
+    }
+    
+    let message = `Count is :: ${count}`;
+    
+    function log() 
+    {
+        console.log(message);
+    }
+
+    return[increment,log];
+}
+
+const [increment,log] = createIncrement();
+increment();
+increment();
+increment();
+log();// What is logged?
+
+
+//output is 0
+
+//because the count is referring to global count which is Zero
```

Comment:
As discussed, this is because your closure forms the message is evaluated already.